src/components/molecules/Card/Card.tsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react'
+import styled from 'styled-components'
+
+// A T O M S
+import { Icon, IconName, IconSize } from '../../atoms/Icon/Icon'
+
+export interface CardProps {
+  className?: string
+  footer?: React.ReactNode
+  icon?: IconName
+  iconSize?: IconSize
+  imageUrl?: string
+  imageMargin?: boolean
+  linkContainerMinHeight?: string
```

Comment:
It's hard to deduce here what you should write to adapt the height of the `linkContainer`. Wouldn't be easier to just have a `.link-container` DOM wrapper element that the developer can adapt with pure css?